NpmAddDependencies.js

```diff
@@ -15,9 +15,20 @@ class NpmAddDependencies {
       if (val && index !== 0 && index !== 1) {
         switch (val) {
           case '--dev':
+          case '--save-dev':
+          case '-D': {
+            this.target = 'devDependencies';
+            break;
+          }
           case '--peer':
-          case '--optional': {
-            this.target = `${val.substring(2)}Dependencies`;
+          case '--save-peer': {
+            this.target = 'peerDependencies';
+            break;
+          }
+          case '--optional':
+          case '--save-optional':
+          case '-O': {
```

Comment:
Since we are now introducing the shortcuts for flags, do we need to change the regex in the `default` switch section to sth like `/^-/`?

Reply:
@arfeo Yes! Very good catch. Done in 0d77a82b5f5f2c8551da6d14da6da2bd8f53e24e.

Reply:
@arfeo I've filed issue #16 to refactor to a 3rd party args parser like `yargs`.